src/main/java/org/fisco/bcos/sdk/channel/ChannelHandlerContextHelper.java

```diff
@@ -0,0 +1,67 @@
+package org.fisco.bcos.sdk.channel;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.util.AttributeKey;
+import org.fisco.bcos.sdk.channel.model.ChannelProtocol;
+import org.fisco.bcos.sdk.channel.model.EnumChannelProtocolVersion;
+import org.fisco.bcos.sdk.channel.model.EnumSocketChannelAttributeKey;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ChannelHandlerContextHelper {
```

Comment:
The class has a better name?

Reply:
VersionNegotiateHelper ?

Reply:
ChannelHandlerContextHelper ->ChannelVersionNegotiation